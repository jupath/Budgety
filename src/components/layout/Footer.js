import React from 'react';
import { Link } from 'react-router-dom';

const Footer = () => (
  <footer className="footer text-center py-4">
    Created by Attila Taligas, view source on <Link to="https://github.com/jupath/Budgety" target="_blank">GitHub</Link>
  </footer>
);

export default Footer;
