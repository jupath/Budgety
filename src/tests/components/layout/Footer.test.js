import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/layout/Footer';

test('should render Footer correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
