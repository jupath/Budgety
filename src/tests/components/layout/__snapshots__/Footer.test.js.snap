// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<footer
  className="footer text-center py-4"
>
  Created by Attila Taligas, view source on 
  <Link
    replace={false}
    target="_blank"
    to="https://github.com/jupath/Budgety"
  >
    GitHub
  </Link>
</footer>
`;
